cmd/auth-handler.go

```diff
@@ -154,6 +154,13 @@ func validateAdminSignature(ctx context.Context, r *http.Request, region string)
 		return cred, owner, s3Err
 	}
 
+	logger.GetReqInfo(ctx).Cred = auth.Credentials{
+		AccessKey:  cred.AccessKey,
+		ParentUser: cred.ParentUser,
+	}
+	logger.GetReqInfo(ctx).Owner = owner
+	logger.GetReqInfo(ctx).Region = region
```

Comment:
do we need region everywhere? is it redundant

Reply:
we only use it in`authorizeRequest` on `auth-handler.go` but I'm not sure it's being read, since that calls comes after `checkRequestAuthTypeCredential` and that one doesn't attach the region